src/com/xilinx/rapidwright/design/AbstractModuleInst.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2021 Xilinx, Inc.
+ * All rights reserved.
+ *
+ * Author: Jakob Wenzel, Xilinx Research Labs.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.xilinx.rapidwright.design;
+
+import com.xilinx.rapidwright.edif.EDIFCell;
+import com.xilinx.rapidwright.edif.EDIFCellInst;
+import com.xilinx.rapidwright.edif.EDIFNet;
+import com.xilinx.rapidwright.edif.EDIFPort;
+
+/**
+ * Abstract base class for {@link ModuleInst} and {@link ModuleImplsInstance}/{@link ModuleImplsInst}
```

Comment:
Removing the pre-rename suggestion.
```suggestion
 * Abstract base class for {@link ModuleInst} and {@link ModuleImplsInst}
```